docs/safetensors.md

```diff
@@ -0,0 +1,136 @@
+# SafeTensors Reader User Guide
+
+The SafeTensors reader in VirtualiZarr allows you to reference tensors stored in SafeTensors files. This guide explains how to use the reader effectively.
+
+## What is SafeTensors Format?
+
+SafeTensors is a file format for storing tensors (multidimensional arrays) that offers several advantages:
+- Safe: No use of pickle, eliminating security concerns
+- Efficient: Zero-copy access for fast loading
+- Simple: Straightforward binary format with JSON header
+- Language-agnostic: Available across Python, Rust, C++, and JavaScript
```

Comment:
Maybe just note here that safetensors was developed by HuggingFace for use in ML model weights. AFAIK no other community uses the format even though they technically could.

Reply:
Added a mention to HuggingFace.